### PR TITLE
Add sort, to_pandas, from_pandas in table

### PR DESCRIPTION
**Pull request by songhappy**

### CODE REVIEW

File: pyzoo/zoo/friesian/feature/table.py [hunk 1]
```diff
@@ -658,6 +658,34 @@ def col(self, name):
...
+    @staticmethod
+    def from_pandas(pandas_df):
+        """
+        Returns the contents of this :class:`pandas.DataFrame` as Table
+        :param pandas_df: pandas dataframe
+        """
+        spark = OrcaContext.get_spark_session()
+        spark.conf.set("spark.sql.execution.arrow.enabled", "true")
+        sparkDF = spark.createDataFrame(pandas_df)
+        return Table(sparkDF)
```
Comment (jason-dai) [inline pyzoo/zoo/friesian/feature/table.py:688]: `self._clone(sparkDF)`

Comment (songhappy) [reply pyzoo/zoo/friesian/feature/table.py:688]: > self._clone(sparkDF)

there is no self in this method, cause it has to have a sparkDF to init Table, current implementation used a static method, similar to read_parquet()

**Comment by songhappy** [comment]
http://10.112.231.51:18888/view/ZOO-PR/job/ZOO-PR-Validation/6297/
failure is irrelevant to this PR

File: pyzoo/zoo/friesian/feature/table.py (hunk 1 above)
Comment (jason-dai) [reply pyzoo/zoo/friesian/feature/table.py:688]: Then you should move this to `FeatureTable` and return a `FeatureTable` instead; it does not make sense to return a base class `Table`

**Pull request merged**